src/kontrol/__main__.py

```diff
@@ -90,9 +90,19 @@ def _ignore_arg(args: dict[str, Any], arg: str, cli_option: str) -> None:
         args.pop(arg)
 
 
-def _load_foundry(foundry_root: Path, bug_report: BugReport | None = None, use_hex_encoding: bool = False) -> Foundry:
+def _load_foundry(
+    foundry_root: Path,
+    bug_report: BugReport | None = None,
+    use_hex_encoding: bool = False,
+    add_enum_constraints: bool = True,
```

Comment:
Is there a reason for making it `True` by default here, while the `--enum-constraints`' default is None?
```suggestion
    add_enum_constraints: bool = True,
```

Reply:
Fixed.